models/cooperation.js

```diff
@@ -0,0 +1,38 @@
+const { Schema, model } = require('mongoose')
+const {
+  enums: { COOPERATION_STATUS }
+} = require('~/consts/validation')
+
+const cooperationSchema = new Schema(
+  {
+    offerId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Offer',
+      required: true
+    },
+    tutorId: {
+      type: Schema.Types.ObjectId,
+      ref: 'User',
+      required: true
+    },
+    studentId: {
+      type: Schema.Types.ObjectId,
+      ref: 'User',
+      required: true
+    },
+    price: {
+      type: Number,
+      required: [true, 'This field cannot be empty.'],
+      min: [1, 'Price must be positive number']
+    },
+    status: {
+      type: String,
+      enum: COOPERATION_STATUS,
+      default: COOPERATION_STATUS[2],
+      required: [true, 'Cooperation status must be selected.']
```

Comment:
remove that req, coz nobody selects it upon creating a cooperation & you do not need status to create a cooperation, you have a default option for it

but add proper validation message for ENUM, what options it can have to choose from, it will be useful for update